src/reporting/Branch.sol

```diff
@@ -117,7 +118,7 @@ contract Branch is DelegationTarget, Typed, Initializable, IBranch {
         return getReportingWindowByTimestamp(block.timestamp + getReportingPeriodDurationInSeconds());
     }
 
-    function getChildBranch(bytes32 _parentPayoutDistributionHash) public returns (IBranch) {
+    function publicGetChildBranch(bytes32 _parentPayoutDistributionHash) public returns (IBranch) {
```

Comment:
Recommend a name that more clearly indicates what is happening under the hood.  Maybe `getOrCreateChildBranch`?